src/frontend/sider/Fagsak/Fagsaklinje/Behandlingsmeny/EndreBehandlendeEnhet/EndreBehandlendeEnhet.tsx

```diff
@@ -34,13 +34,11 @@ const EndreBehandlendeEnhet: React.FC = () => {
     };
 
     const erLesevisningPåBehandling = () => {
-        const åpenBehandlingData = hentDataFraRessurs(åpenBehandling);
-        const steg = åpenBehandlingData?.steg;
+        const steg = behandling?.steg;
```

Comment:
Tror ikke du trenger `?` her

```suggestion
        const steg = behandling.steg;
```